src/ol/render/webgl/MixedGeometryBatch.js

```diff
@@ -70,6 +71,26 @@ import {inflateEnds} from '../../geom/flat/orient.js';
  */
 class MixedGeometryBatch {
   constructor() {
+    this.globalCounter = 0;
+    /**
+     * Refs are used as keys for hit detection.
+     * @type {Map<number, Feature|RenderFeature>}
```

Comment:
It looks like this and the properties below could be `@private` so other parts of the library don't mistakenly think they can mess with them.